app/serializers/user_card_serializer.rb

```diff
@@ -224,7 +224,7 @@ def card_background_upload_url
   end
 
   def include_status?
-    SiteSetting.enable_user_status
+    SiteSetting.enable_user_status && user.user_status && !user.user_status.expired?
```

Comment:
I would extract this into a method into the `User` class instead of duplicating the logic between two serializers. Perhaps something like `user.has_status?`